addons/mail/static/tests/owl/component/discuss_tests.js

```diff
@@ -4014,34 +4008,34 @@ QUnit.test('composer state: attachments save and restore', async function (asser
     assert.strictEqual(
         document
             .querySelectorAll(`
-            .o_Composer_attachmentList
-            .o_Attachment`)
+                .o_Composer_attachmentList
+                .o_Attachment`)
             .length,
         3,
         "should have 1 attachment in the composer");
     assert.strictEqual(
         document
             .querySelectorAll(`
-            .o_Composer_attachmentList
-            .o_Attachment`)[0]
+                .o_Composer_attachmentList
+                .o_Attachment`)[0]
             .dataset
             .attachmentLocalId,
         'ir.attachment_2',
         "should have ir.attachment_2 attachment");
```

Comment:
_"... as the 1st attachment"
"... as the 2nd attachment"
"... as the 3rd attachment"_

Assertions seem to also check position of attachments.